api/app/signals/apps/dataset/management/commands/load_gebieden.py

```diff
@@ -0,0 +1,93 @@
+import requests
+from django.contrib.gis.geos import LinearRing, MultiPolygon, Polygon
+from django.core.management import BaseCommand
+
+from signals.apps.signals.models import Area, AreaType
+
+
+class GebiedenAPIGeometryLoader:
```

Comment:
This should have its own file and should not be part of the management command file.

Reply:
agree, and fixed